package.json

```diff
@@ -24,10 +24,10 @@
       "rollup -c && rollup -c rollup.browser.config.js && rollup -c rollup.test-utils.config.js",
     "watch": "tsc -w",
     "lint": "tslint --project tsconfig.json --config tslint.json",
+    "lint:fix":
+      "npm run prettier && tslint 'src/*.ts*' --project tsconfig.json --fix",
     "lint-staged": "lint-staged",
-    "lint-fix":
-      "npm run pretty && tslint 'src/*.ts*' --project tsconfig.json --fix",
-    "pretty":
```

Comment:
Just a small tip. But could you try and keep your PR as condensed as possible so that you are fixing a single thing in a PR. So I would prefer to see unrelated changes like this in a separate PR. I prefer to keep the PRs as small as possible in order to maximize the quality of the review.